File: examples/example_atexit.mjs
#!/usr/bin/env node
import atexit from "atexit";

const implement = "atexit.mjs";

console.log(`=== ${implement} ===`);

console.log(implement, ": borrow a space ship from 👽");
atexit.register(console.log, implement, ": return a space ship ⏲");

console.log(implement, ": goto mars 🚀");
atexit.register(console.log, implement, ": went back from mars 👋");

console.log(implement, ": one night on mars 🌙");
atexit.register(console.log, implement, ": wakeup on mars 🌞");

File: examples/example_atexit.py
#!/usr/bin/env python3
import atexit

implement = 'atexit.py'

print(f'=== {implement} ===')

print(implement, ': borrow a space ship from 👽')
atexit.register(print, implement, ': return a space ship ⏲')

print(implement, ': goto mars 🚀')
atexit.register(print, implement, ': went back from mars 👋')

print(implement, ': one night on mars 🌙')
atexit.register(print, implement, ': wakeup on mars 🌞')
